# Test try block with no exception thrown
(try
  (println "No exception")
  (catch e
    (println "Should not print")))